src/Generator/UsersGenerator.php

```diff
@@ -105,6 +105,21 @@ public function generateRecord($n)
         $userData = $data['data']['users'][0];
         $fullName = sprintf("'%s %s'", trim($userData['first_name'], "'"), trim($userData['last_name'], "'"));
         $description = sprintf("'Private team for %s'", trim($userData['user_name'], "'"));
+        $managerID = null;
+
+        if ($n % 10 != 0) {
+            $managerID = 'seed-Users' . ($n - ($n % 10));
+        } else {
+            if ($n > 100) {
+                $managerID = 'seed-Users' . ($n - ($n % 100));
+            }
+        }
```

Comment:
Why not just `$managerID = $this->idGenerator->generateTidbitID((int) $n / 10, 'Users')`?